### DISCUSSION TITLE
Request to modify ha-mode policy to quorum queue format policy

### DISCUSSION THREAD

**Discussion by qwq1357** [rabbitmq/rabbitmq-server#12245] [Questions] [answered]
If you are using the previous ha-mode and a message is displayed saying that you are using an unsupported function, can you modify the previous policy to a quorum queue type policy (with the same options)?

![policy](https://github.com/user-attachments/assets/5af95e21-d584-464c-a91f-574aacf80a06)

**Comment by kjnilsson** [comment, marked as answer]
No quorum queues are completely different queue types. Please review the documentation:

https://www.rabbitmq.com/docs/quorum-queues

And the classic to quorum migration document:

https://www.rabbitmq.com/docs/migrate-mcq-to-qq
